test/acceptance/cli-test.js

```diff
@@ -1606,6 +1619,91 @@ describe('ember-template-lint executable', function () {
         expect(result.stdout).toEqual('');
       });
 
+      it('with --update-todo, outputs todos created summary', async function () {
```

Comment:
Can you add a similar test when there are no todos needed?

Reply:
You mean add a test without the `--update-todo` option, and that results in the message _not_ being displayed?

Reply:
Yes, kinda. Though in the other comment thread I agree with you that we should always print some message about todos created/removed whenever `--update-todo` is used. So in that case, the new test should basically just confirm that `--update-todo` with no prior failures results in a message like `0 todos created`.

Reply:
Agree.

Reply:
Done.